lib/watir-webdriver/wait.rb

```diff
@@ -94,16 +94,9 @@ def wait_while(*args, &blk)
     end
   end
 
-  #
-  # Wraps an Element so that any subsequent method calls are
-  # put on hold until the element is present (exists and is visible) on the page.
-  #
-
-  class WhenPresentDecorator
+  class BaseDecorator
     extend Forwardable
 
-    def_delegator :@element, :present?
```

Comment:
Take a look at #216.

I don't think it makes sense to add delegator for `#enabled?`, but `#present?` is definitely needed.